src/components/CategoryBreadcrumb.jsx

```
@@ -0,0 +1,53 @@
+import { Link } from 'react-router-dom';
+import * as NavigationMenu from '@radix-ui/react-navigation-menu';
+// import { ChevronRightIcon } from '@radix-ui/react-icons';
+
+export default function CategoryBreadcrumb({ category, categoryId }) {
```

ESLint

[error] 'categoryId' is defined but never used.

(no-unused-vars)